component/registry/registry.go

```diff
@@ -0,0 +1,63 @@
+package registry
```

Comment:
Wonder if naming this "registry" will lead to confusion with the "docker registry" (for images)